### PR TITLE
Feature Update: Dark Mode

### PR DESCRIPTION
**Pull request by milindvishnoi**

### CODE REVIEW

File: components/global/DarkModeButton.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <Light class="logo light" v-if="isDark" @click="$colorMode.preference = 'light'" />
+    <Dark class="logo" v-else @click="$colorMode.preference = 'dark'" />
+  </div>
+</template>
+
+<script>
+import Light from '../../static/svg/sun.svg?inline'
```
Comment (ArsalaBangash) [inline components/global/DarkModeButton.vue:9]: Can we change the path to @/static?

File: components/global/DarkModeButton.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        colors: ['light', 'system', 'dark']
+      }
+  },
+  computed: {
+    isDark() {
+      return this.$nuxt.$colorMode.value == 'dark'
+    }
+  },
+  components: {
+    Light,
```
Comment (ArsalaBangash) [inline components/global/DarkModeButton.vue:24]: Let's call these LightIcon and DarkIcon

File: components/global/DarkModeButton.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    Dark
+  }
+}
+</script>
+
+<style scoped>
+.logo {
+  width: 35px;
```
Comment (ArsalaBangash) [inline components/global/DarkModeButton.vue:32]: 36px since even numbers and multiples of 12 are more commonly used.

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]
Solid iteration!

**Comment by hiimchrislim** [comment]
image
It seems like a few of our fonts and colors aren't applied

**Comment by hiimchrislim** [comment]
The search bar has 2 colors and the text should perhaps be a dark grey
image

**Comment by hiimchrislim** [comment]
image

If possible, could we use the same color for the sidebar as the one in the (light grey/blue-ish) that says Web Development?

**Comment by hiimchrislim** [comment]
image
The FAQ will also need to have another color as well (where it's easy to see the accordion cards)

**Comment by hiimchrislim** [comment]
image
This is a very minor thing, but is it possible if we could fill in the SVG to a white/grey to it's easier to see the UTMSAM logo?

**Comment by milindvishnoi** [comment]
@hiimchrislim I have made the required changes, please approve this ASAP coz I'll need this PR for my other gpa-calculator PR.

**Comment by hiimchrislim** [comment]
Great job!

**Pull request merged**